octodns_bind/__init__.py

```diff
@@ -243,10 +262,10 @@ def _apply(self, plan):
 
 @ $default_ttl IN SOA $primary_nameserver $hostmaster_email (
     $serial ; Serial
-    3600       ; Refresh (1 hour)
-    600        ; Retry (10 minutes)
-    604800     ; Expire (1 week)
-    3600       ; NXDOMAIN ttl (1 hour)
+    $refresh ; Refresh (1 hour)
```

Comment:
Just a nitpick, if they are able to be changed then we probably don't need the `(1 hour)`, `(1 week)` after them, as that value could be different than what the user specifies.